src/layer/S3OLCIL2CDASLayer.ts

```diff
@@ -1,6 +1,18 @@
 import { DATASET_CDAS_S3OLCIL2 } from './dataset';
-import { S3OLCILayer } from './S3OLCILayer';
+import { S3SLSTRCDASLayer } from './S3SLSTRCDASLayer';
 
-export class S3OLCIL2CDASLayer extends S3OLCILayer {
+import { Link, LinkType } from './const';
+
+export class S3OLCIL2CDASLayer extends S3SLSTRCDASLayer {
```

Comment:
We shouldnt extend from `S3SLSTRCDASLayer`, as S3-SLSTR has different properties than S3-OLCI, for example `S3SLSTRView`. It might be best to just extend from `AbstractSentinelHubV3WithCCLayer`.

Reply:
Agreed and changed